plugins/woocommerce/legacy/js/frontend/single-product.js

```diff
@@ -93,7 +93,6 @@ jQuery( function( $ ) {
 
 		// No images? Abort.
 		if ( 0 === this.$images.length ) {
-			this.$target.css( 'opacity', 1 );
```

Comment:
Maybe we can move this line outside the if to let oldest template to work with this changes.